Tasks/AzureTestPlanV0/task.json

```diff
@@ -14,7 +14,7 @@
   "version": {
     "Major": 0,
     "Minor": 237,
-    "Patch": 0
+    "Patch": 5
```

Comment:
Looks like in last week of sprint, minor version should be updated. Let's update it to 2.238.0

reference: https://github.com/microsoft/azure-pipelines-tasks/pull/19687#discussion_r1538816389

Reply:
Updated